molecule/default/molecule-with-systemd.yml

```diff
@@ -0,0 +1,122 @@
+---
+# This molecule configuration file is suitable for testing Ansible
+# roles that _do_ require SystemD.  If your Ansible role _does not_
+# require SystemD then you should use molecule-no-systemd.yml instead.
+#
+# Note that the molecule configuration file that is symlinked to
+# molecule.yml is the one that will be used.
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint: |
+  set -e
+  yamllint .
+  ansible-lint
+  flake8
+platforms:
+  # - name: amazonlinux2_systemd
```

Comment:
Should this just be removed as you've done with some other roles?

Reply:
I tried to copy what was done in the existing `molecule.yml` file when doing these PRs.  Also, in this case, cyhy-runner will soon become "all but Debian 9" instead of "Debian 9 only" so I don't think it's a bad idea to keep them around.